src/components/tx/TxSimulation/SimulationResult.tsx

```diff
@@ -44,8 +58,18 @@ export const SimulationResult = ({
           </Typography>
         ) : (
           <Typography>
-            The transaction failed during the simulation throwing error <b>{simulation?.transaction.error_message}</b>{' '}
-            in the contract at <b>{simulation?.transaction.error_info?.address}</b>. Full simulation report is available{' '}
+            The transaction failed during the simulation{' '}
+            {isCallTraceError ? (
+              <>
+                with error <b>{callTraceErrors[0].error}</b>.
```

Comment:
I think this leads to too technical errors. The one in the PRs screenshot for instance reads "value: execution error" which does not really help users IMO.

I would just scan for the event log and give a generic error here OR if we want to give more details on what failed, we could start collecting common `call_trace` errors and map them to human readable error messages. 

For the out of gas error for instance this error is present in the call traces:
`"error": "out of gas"` which we could rephrase to "The transaction failed with an out of gas error. Try increasing the  safeTxGas and try again."

Reply:
For future reference: we've generalised the error messages in accordance with the above clarification.